src/rulesets/Base/base.jl

```diff
@@ -83,14 +83,20 @@
 
 @scalar_rule(hypot(x, y), (x / Ω, y / Ω))
 @scalar_rule(sincos(x), @setup((sinx, cosx) = Ω), cosx, -sinx)
-@scalar_rule(atan(x, y), @setup(u = x^2 + y^2), (y / u, -x / u))
```

Comment:
this is jsut renaming x -> y, y -> x, right?

Reply:
Yes, that corresponds better to the geometric meaning